sql-plugin/pom.xml

```diff
@@ -97,6 +97,13 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.nvidia</groupId>
+            <artifactId>rapids-4-spark-hybrid_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
```

Comment:
Instead of using the rapids plugin version,  rapids-4-spark-hybrid dependency should have a separate version definition

I'll follow it up.